### PR TITLE
fix(DB/Creature): Silvermoon Pet Trainer

### PR DESCRIPTION
**Pull request by Knindzagxg**
Transfering from SC by @FrancescoBorzi request.

Update and corrections for Silvermoon Pet Trainer.

### CODE REVIEW

**Comment by Geekena** [comment]
tested
no error or problems

**Review by FrancescoBorzi — APPROVED** [review]

**Review by Kitzunu — CHANGES_REQUESTED** [review]
This GUID is already IN USE by another Creature!

Please make sure to always use a FREE GUID when you add new creatures.

creature
---
```sql
SELECT * FROM `creature` WHERE guid = 248570
```
| guid | id | map | zoneId | areaId | spawnMask | phaseMask | modelid | equipment_id | position_x | position_y | position_z | orientation | spawntimesecs | wander_distance | currentwaypoint | curhealth | curmana | MovementType | npcflag | unit_flags | dynamicflags | ScriptName | VerifiedBuild | 
| ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | ---: | --- | ---: | 
| 248570 | 32803 | 571 | 0 | 0 | 1 | 1 | 0 | 0 | 3936,92 | -595,444 | 241,153 | 5,95157 | 120 | 0 | 0 | 1 | 0 | 0 | 0 | 0 | 0 |  | 0 |

**Comment by Knindzagxg** [comment]
Sorted xP

File: data/sql/updates/pending_db_world/rev_1598881374545560300.sql [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* Content 3.3.5 */
+
+-- NPC entry 40405 Kieupid, Pet Trainer in Silvermoon City (map: 530, zone ID 0, area ID 0)
+SET @CGUID = 248600;
+
+DELETE FROM `creature` WHERE `id` = 40405;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1598881374545560300.sql:13]:
```suggestion
DELETE FROM `creature` WHERE `id` = 40405 AND `guid` = @CGUID;
```

You ALWAYS need to delete the Primary Key.

**Comment by FrancescoBorzi** [comment]
@Kitzunu error:

```
ERROR 1054 (42S22) at line 112550: Unknown column 'spawndist' in 'field list'
```

@Geekena on which AC version are you testing?

**Comment by Kitzunu** [comment]
Spawndist hasnt existed for MONTHS

**Comment by Kitzunu** [comment]
It is wanderdistance/wander_distance I cant remember

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/commit/8a1eab2c2330537a36e1f1fdae56ea9ad40b5e43 April

**Comment by Knindzagxg** [comment]
Just saw mate, had no time before :S
My core is updated, i just xfered from SC link you gave.
Havent tested myself sadly.

Gonna sort in next few, just came home xP

**Comment by Knindzagxg** [comment]
SQL sorted now, gonna test it as well.

**Comment by Knindzagxg** [comment]
Ok, it seems someone moved this and Dalaran fountain before but havent marked as done.

Its double spawn again, and i think there is no this for this and Dalaran F. PRs.
Can someone check and confirm double spawns so we can close?

**Comment by FrancescoBorzi** [comment]
closing as discussed on Discord

**Pull request closed without merging**